addon/src/settled.ts

```diff
@@ -2,9 +2,9 @@
 // "provides" this public API, but does not for earlier versions. As a result,
 // this type will be `any`.
 import { _backburner } from '@ember/runloop';
-import Ember from 'ember';
-
-import EmberApplicationInstance from '@ember/application/instance';
+import { Test } from 'ember-testing';
+export const checkWaiters = Test.checkWaiters;
```

Comment:
Why is this being exported? This seems to have caused a type regression, see https://github.com/emberjs/ember-test-helpers/issues/1502

Reply:
great question -- I had originally thought that the original implementation was exported -- but I see in this diff that it is not. let's see if not exporting it resolves that issue